src/openrct2/ride/VehiclePaint.cpp

```diff
@@ -3663,7 +3684,16 @@ void VehicleVisualDefault(
 {
     if (vehicle->Pitch < std::size(PaintFunctionsByPitch))
     {
-        PaintFunctionsByPitch[vehicle->Pitch](session, vehicle, imageDirection, z, carEntry);
+        if (vehicle->IsReversed)
+        {
+            auto imagePitch = PitchInvertTable[vehicle->Pitch];
+            auto imageYaw = (imageDirection + (OpenRCT2::Entity::Yaw::BaseRotation / 2)) % OpenRCT2::Entity::Yaw::BaseRotation;
```

Comment:
Should this use modulo?

Reply:
Changed to use a bit mask - I'm assuming the yaw base will always be a power of two